### DISCUSSION TITLE
A well-done Redash fork can be used as a reference.

### DISCUSSION THREAD

**Discussion by Jengro777** [RedashCommunity/redash#38] [General]
refer to： https://github.com/dazdata/redash

Official website：https://redash.dazdata.com/

However, this mainly promotes the commercial version, and in terms of openness, it cannot compare to our community version of Redash.

**Comment by justinclift** [comment]
What seems to be better with the Dazdata fork?  When I looked at it briefly a few weeks ago, they haven't improved the dependency versions so they likely have the same security vulnerabilities as the main getredash repo.

**Reply by Jengro777** [reply]
Perhaps the commercial version of Dazdata did better, while the open source version did not do more

**Reply by justinclift** [reply]
Is there evidence they've changed anything apart from making a translated website? :smile:

**Reply by Jengro777** [reply]
The commercial version of Daztada does indeed offer many additional features. As someone based in China, I happen to know some of the employees working for Daztada personally, and they have made significant improvements to Redash's commercial version that suit China's national conditions and requirements.

**Reply by justinclift** [reply]
Cool.  Can you ask them if they've updated the dependencies in their commercial version to no longer have security vulnerabilities?

**Reply by Jengro777** [reply]
I consulted with them, and the commercial version has made some security vulnerability fixes, but not all of them.

To be precise, they have fixed all the critical vulnerabilities. The less severe ones have not been addressed.

**Reply by justinclift** [reply]
> To be precise, they have fixed all the critical vulnerabilities.

Cool.  Would you be ok to ask them if they could share those fixes with us?

**Reply by Jengro777** [reply]
They operate commercially, so everything they can share is in their Redash fork.

However, I think contacting them directly may yield different results.
